composer/utils/file_helpers.py

```diff
@@ -387,10 +389,11 @@ def maybe_create_remote_uploader_downloader_from_uri(
     elif backend == 'wandb':
         raise NotImplementedError(f'There is no implementation for WandB via URI. Please use '
                                   'WandBLogger with log_artifacts set to True')
-
+    elif backend == 'dbfs' and path.startswith('Volumes/'):
+        return RemoteUploaderDownloader(bucket_uri=f'{backend}://{bucket_name}', backend_kwargs={uri: uri})
```

Comment:
bucket name here is empty string right? should we just leave it out of the format string for clarity or is there a reason it is there?